src/components/newsletter/index.js

```diff
@@ -67,10 +72,42 @@ const Newsletter = () => {
         return await response.json()
       }
 
-      postData(GATSBY_NEWSLETTER_BACKEND_URL, {
-        email: email,
-        tag: tag,
-      })
+      const data = {
+        submittedAt: new Date().getTime(),
+        fields: [
+          {
+            name: "email",
+            value: email,
+          },
+        ],
+        context: {
+          pageUri: window.location.href,
+          hutk: cookies.hubspotutk || ''
+        },
+        legalConsentOptions: {
+          consent: {
+            consentToProcess: true,
+            text:
+              "I agree to allow MayaData to store and process my personal data.",
```

Comment:
We can't have company-specific handles into this website. The subscription should ideally add the users to the CNCF users list and then send the news letters from there.